applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -661,14 +661,16 @@ int le_audio_user_defined_button_press(enum le_audio_user_defined_action action)
 	return 0;
 }
 
-int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate)
+int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate, uint32_t *octets_per_frame)
 {
 	/* Get the configuration for the sink stream */
 	int frames_per_sec = 1000000 / bt_codec_cfg_get_frame_duration_us(audio_streams[0].codec);
 	int bits_per_frame = bt_codec_cfg_get_octets_per_frame(audio_streams[0].codec) * 8;
 
 	*sampling_rate = bt_codec_cfg_get_freq(audio_streams[0].codec);
 	*bitrate = frames_per_sec * bits_per_frame;
+	*octets_per_frame = bits_per_frame / 8;
```

Comment:
This can be done simpler. On line 668 call this variable octets_per_frame. Here it is first multiplied by 8 and then divided by 8 :)

Reply:
Added changes